### DISCUSSION TITLE
Star ratings

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3932] [Ideas] [answered]
We are starting to create survey forms that include star ratings, rather than the standard radio button.  Is there a Paste pattern for this?  We found the star icon, but there's not really a filled/partially filled option for these.

See example below:
![image](https://github.com/twilio-labs/paste/assets/9394413/34d8a432-494e-461b-abad-e4aeeee3e490)

**Comment by serifluous** [comment]
Hey @tonge495 !

For a 5-point rating, customers have to quickly differentiate what a 4 vs. 5 (for example) or 1 vs. 2 means to them for this experience, which could end up giving you fuzzier results compared to a simpler sentiment rating.  For this survey, is your team hoping to get any benefit out of using a 5-point rating, vs a 2- or 3-point sentiment rating (like thumb up / thumb down or [happy/neutral/sad](https://github.com/twilio-labs/paste/discussions/3712))?

**Reply by tonge495** [reply]
The above screenshot is from the CDE team and a similar survey on Help Center is for the CSAT survey, which has a 5 point rating (Very dissatisfied to very satisfied).  One of the PMs on CDE was suggesting for the CSAT survey for Help Center to use the same pattern as above instead of radio options, which was why this was brought up.  Jiyeon was planning to start a dependency intake form for the filled version.

**Reply by serifluous** [reply]
Thanks for the dependency request! Ok cool, that makes sense for CSAT. We'll add it to our next round of icon additions.

I also checked with Research & they suggested for star ratings on CSAT to:

1. Specifically ask about "satisfaction" in the rating question, e.g., "How would you rate your satisfaction with x?"
2. Adding "Very dissatisfied" and "Very satisfied" to the ends of the stars can help with clarity (but not necessary)

**Comment by serifluous** [comment, marked as answer]
Hey @tonge495  ! The icon is now available in Paste. Check it out here: https://paste.twilio.design/components/icon
